docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+# Deploying OpenShift bare-metal workers on OpenStack cloud provider
+TBD introduction
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Conciderations whe deploying bare-metal workers](#conciderations-whe-deploying-bare-metal-workers)
+- [Deploying cluster with BM workers on tenant network deployed by the installer](#deploying-cluster-with-bm-workers-on-tenant-network-deployed-by-the-installer)
+- [Deploying cluster with BM workers on preexisting network](#deploying-cluster-with-bm-workers-on-preexisting-network)
+- [Deploy cluster with vm workers only and add BM workers after](#deploy-cluster-with-vm-workers-only-and-add-bm-workers-after)
+
+
+## Prerequisites
+* [Ironic bare metal service][2] is enabled an accessible through the [OpenStack Compute API][3].
+* Bare-metal workers are available as an [OpenStack flavor][1]
+* The image used to boot the bare-metal worker is available from the [Glance OpenStack image service][4] 
+* The cloud provider is configured such that bare-metal and vms can be attached to the same tenant network.
```

Comment:
"bare-metal and vms can be attached to the same tenant network" is only needed for one case, right? By finding this prerequisite here, it looks like it's a prerequisite for BM workers in general.

IIUC, the two next prerequisites are prerequisites for two other cases. By seeing them all together, I might think that I need to have them all at the same time

Reply:
three  cases which require same BMs and vms to be connected to same subnet (need to clarify RFE if not). 
case 1: https://github.com/openshift/enhancements/pull/146/files#diff-1e496c947a523fc98e7acd29798b1b5aR148
case 2: https://github.com/openshift/enhancements/pull/146/files#diff-1e496c947a523fc98e7acd29798b1b5aR148